recipes/tsne/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "tsne" %}
+{% set version = "v0.1.9" %}
+{% set sha256 = "1d41ebfc24494f949d87bd137eb68760179db80f6b70ac7be309dc93f9a2b296" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/WiscEvan/tsne/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  skip: false  # [py==27]
```

I think that should be then:
```suggestion
  skip: true  # [py3k and py<37]
```